### PR TITLE
fix:add flake8/black on precommit hook

### PR DESCRIPTION
**Pull request by kirtimanmishrazipstack**
## What

- Add flake8 on pre-commit hook

## Why

- it was commented on this repo

## How

- Added on `pre-commit-config.yaml`

## Database Migrations

- N/A

## Env Config

- N/A

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by ritwik-g — CHANGES_REQUESTED** [review]
@kirtimanmishrazipstack please change max length in black as well

**Comment by kirtimanmishrazipstack** [comment]
> @kirtimanmishrazipstack please change max length in black as well

done @ritwik-g

**Review by ritwik-g — APPROVED** [review]

**Review by chandrasekharan-zipstack — CHANGES_REQUESTED** [review]
Please update pyproject.toml as well and re-run `pre-commit run --all-files`

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -50,23 +50,23 @@ repos:
     rev: 24.3.0
     hooks:
       - id: black
-        args: [--config=pyproject.toml, -l 80]
+        args: [--config=pyproject.toml, -l 88]
```
[the remark this answers is not in the document]
Comment (kirtimanmishrazipstack) [reply .pre-commit-config.yaml:53]: done @chandrasekharan-zipstack

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM

**Pull request closed without merging**